### ISSUE TITLE
Implement inverse power law; distance simulations

### ISSUE THREAD

**Issue by UlascanErsoy** [UlascanErsoy/Team-7---Multimodal-Accessibility#25]
We must implement inverse power law distance simulations into the `SpatialPanner` object.

```python
class SpatialPanner(Effect):
    """Main page of the application."""

    def __init__(self, azimuth: float, elevation: float, distance: float, distance_model: Optional[DistanceModel]):
        """
        :param azimuth: azimuth of the direction angles (think l / r)
        :type azimuth: float
        :param elevation: elevation of the direction in angles
        :type azimuth: float
        """
```

We should have a separate class to store the PowerLaw model. This way we can provide multiple options and models (in the future) using dependency injection.

**Comment by UlascanErsoy** [comment]
Working on this.

**Comment by UlascanErsoy** [comment]
Merged with dev as of b4082017a8220d82d15277588f21514b7a9c9e2d
